meinberlin/templates/base.html

```diff
@@ -13,6 +13,16 @@
     <script src="{% static 'vendor.js' %}"></script>
     <script src="{% static 'meinberlin.js' %}"></script>
     <meta name="viewport" content="width=device-width" />
+    {% if request.user.is_authenticated %}
+    <script>
+        if (window.opener) {
+            window.opener.postMessage(
+                JSON.stringify({name: "popup-close",}),
+                location.origin
+            )
+        }
```

Comment:
This behavior should be document as a general restriction: Opening meinberlin in a popup for anything other than login is not supported.